src/edu/washington/escience/myria/expression/PyUDFExpression.java

```diff
@@ -0,0 +1,77 @@
+package edu.washington.escience.myria.expression;
+
+import java.util.List;
+
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import edu.washington.escience.myria.Type;
+import edu.washington.escience.myria.expression.evaluate.ExpressionOperatorParameter;
+
+/**
+ *
+ * Python expressions.
+ *
+ */
+public class PyUDFExpression extends NAryExpression {
+  /***/
+  private static final long serialVersionUID = 1L;
+  /** The logger for this class. */
+  private static final org.slf4j.Logger LOGGER =
+      org.slf4j.LoggerFactory.getLogger(PyUDFExpression.class);
+
+  /** The name of the python function. */
+  @JsonProperty private final String name;
+
+  /**
+   * Output Type of python function.
+   */
+  @JsonProperty private final Type outputType;
+
+  /**
+   * This is not really unused, it's used automagically by Jackson deserialization.
+   */
+  @SuppressWarnings("unused")
+  private PyUDFExpression() {
+    name = "";
+    //set default return type this is changed once pythonfunction is retrived.
+    outputType = Type.BLOB_TYPE;
+  }
```

Comment:
this method is always called when expression is initialized, and I need to set default type here.